lucet-runtime/lucet-runtime-internals/src/instance/signals.rs

```diff
@@ -164,6 +166,13 @@ extern "C" fn handle_signal(signum: c_int, siginfo_ptr: *mut siginfo_t, ucontext
                 .as_mut()
         };
 
+        if signal == Signal::SIGALRM {
```

Comment:
We could `assert!(!inst.kill_state.terminable.load(Ordering::SeqCst))` here, just because an alarm without marking the instance unterminable would be an error, but I'm not very opinionated on this. I'm pretty sure it would just serve to make a spurious sigalrm into a host-crashing assert fail.

Reply:
Let's do a `debug_assert!` for now, and remember to log it properly later when we have some sort of stats interface